packages/terra-form-select/src/_Frame.jsx

```diff
@@ -166,7 +166,7 @@ class Frame extends React.Component {
       case Variants.COMBOBOX:
         return <input {...inputAttrs} value={hasSearchChanged ? searchValue : display} />;
       default:
-        return display || <div className={cx('placeholder')}>{placeholder}</div>;
+        return display || <div className={cx('placeholder')}>{placeholder || '\xa0'}</div>;
```

Comment:
This will prevent the placeholder from collapsing if an empty string was provided as a placeholder. This is the only code component source change in this pull request. I can split this change out into another PR if need be.